src/Microsoft.ML.OnnxTransformer/OnnxUtils.cs

```diff
@@ -167,29 +169,47 @@ public OnnxVariableInfo(string name, OnnxShape shape, Type typeInOnnxRuntime, Da
         /// <param name="recursionLimit">Optional, specifies the Protobuf CodedInputStream recursion limit. Default value is 100.</param>
         /// <param name="interOpNumThreads">Controls the number of threads used to parallelize the execution of the graph (across nodes).</param>
         /// <param name="intraOpNumThreads">Controls the number of threads to use to run the model.</param>
-        public OnnxModel(string modelFile, int? gpuDeviceId = null, bool fallbackToCpu = false,
+        public OnnxModel(IHostEnvironment env, string modelFile, int? gpuDeviceId = null, bool fallbackToCpu = false,
```

Comment:
> public OnnxModel(IHostEnvironment env, string modelFile, int? [](http://example.com/codeflow?start=7&length=62)

I assume there is no app compat concern changing the public API. right?

Reply:
Nope, cause the OnnxModel class is an internal class only so we can do whatever we want with it.